### PR TITLE
Added Register User Pages

### PR DESCRIPTION
**Pull request by kevinburnside**
Added register pages along with paymentinfo pages. 
data is being passed from register to payment pages.
Still need to create the data property to send info to the backend.

### CODE REVIEW

File: Keanu/src/pages/login/login.html [hunk 1]
```diff
@@ -30,4 +30,7 @@
   <div padding>
     <button block ion-button (click)="login($event)" >Login</button>
   </div>
+  <div padding>
```
Comment (aagavin) [inline Keanu/src/pages/login/login.html]: Is it possible to make this button less prominent or smaller?
Also can we add a message that says something like "don't have an account? Register here"

File: Keanu/src/pages/paymentInfo/paymentInfo.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <ion-item>
+      <ion-label floating>Credit Card Expiry Date</ion-label>
+      <ion-input type="text" [(ngModel)]="cardExpiry"></ion-input>
+    </ion-item>
+    </div> 
+  </ion-list>
+  <div padding>
+    <button block ion-button>Next</button>
+  </div>
+</ion-content>
```
Comment (aagavin) [inline Keanu/src/pages/paymentInfo/paymentInfo.html]: new line at end of file needed

File: Keanu/src/pages/register/register.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+     
+    <ion-item>
+      <ion-label floating>Password</ion-label>
+      <ion-input type="password" [(ngModel)]="password"></ion-input>
+    </ion-item>  
+  </ion-list>
+  <div padding>
+    <button block ion-button (click)="next($event)" >Next</button>
+  </div>
+</ion-content>
```
Comment (aagavin) [inline Keanu/src/pages/register/register.html]: new line at end of file needed

**Review by aagavin — CHANGES_REQUESTED** [review]

**Review by MavOmidK — APPROVED** [review]
Looks good, all works

**Review by aagavin — APPROVED** [review]

**Pull request merged**